controllers/sr_controller/sr/robot/robot.py

```diff
@@ -61,7 +61,7 @@ def webots_step_and_should_continue(self, duration_ms: int) -> bool:
         """
         Run a webots step of the given duration in milliseconds.
 
-        Returns whether or not Webots is about to terminate the simulation.
+        Returns whether or not the simulation should continue (ie is webots terminating).
```

Comment:
```suggestion
        Returns whether or not the simulation should continue (based on
        Webots telling us whether or not the simulation is about to end).
```

Reply:
Or even just drop the explanation? The inline comment somewhat covers that for anyone who cares.